app/helpers/raster/georaster.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from os.path import dirname
 import logging
-from struct import unpack
+from struct import unpack  # pylint: disable=no-name-in-module
```

Comment:
The pylint pragma struct no-member seesms to me very strange, at least on don't see on my locals pylint errors with struct ? `struct` being a python builtin library it should not have such linting issue, we need to check together what is the issue here !

Reply:
Well, I added them because I saw them locally (and linting was failing on CI), but we could investigate and check this out.